seaborn/matrix.py

```diff
@@ -130,12 +130,22 @@ def __init__(self, data, vmin, vmax, cmap, center, robust, annot, fmt,
 
         # Get the positions and used label for the ticks
         nx, ny = data.T.shape
-        xstart, xend, xstep = 0, nx, xtickevery
-        self.xticks = np.arange(xstart, xend, xstep) + .5
-        self.xticklabels = xticklabels[xstart:xend:xstep]
-        ystart, yend, ystep = (ny - 1) % ytickevery, ny, ytickevery
-        self.yticks = np.arange(ystart, yend, ystep) + .5
-        self.yticklabels = yticklabels[ystart:yend:ystep]
+
+        if xticklabels == []:
```

Comment:
`if not xticklabels` fails if `xticklabels` is a numpy array. Something about truth value being ambiguous for numpy arrays.